.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "gitlens.currentLine.enabled": false
+}
```

Comment:
generally you should prefer having a trailing newline, to prevent tools like github from giving you angry red circles.